src/js/shariff.js

```diff
@@ -94,15 +85,23 @@ const Defaults = {
   },
 }
 
-class Shariff {
+export class Shariff {
   constructor(element, options) {
     // the DOM element that will contain the buttons
     this.element = element
 
-    // Ensure elemnt is empty
-    $(element).empty()
-
-    this.options = $.extend({}, Defaults, options, $(element).data())
+    this.options = Defaults;
```

Comment:
```suggestion
    this.options = Object.assign({}, Defaults);
```
Clone the Default object so it's not modified later.
This fixes the issue that wrong defaults are used when having more than one Shariff instances on a page.
E.g. when running the demo, wrong button styles "icon" are used as defaults.